library/core/src/alloc/layout.rs

```diff
@@ -4,14 +4,17 @@
 // collections, resulting in having to optimize down excess IR multiple times.
 // Your performance intuition is useless. Run perf.
 
-use safety::requires;
+use safety::{invariant, requires};
 use crate::error::Error;
 use crate::ptr::{Alignment, NonNull};
 use crate::{assert_unsafe_precondition, cmp, fmt, mem};
 
 #[cfg(kani)]
 use crate::kani;
 
+#[cfg(kani)]
```

Comment:
This doesn't need a config Kani though, right?

Reply:
It does. Without it, the `check_rustc` script fails with an unused import error:

```
error: unused import: `Invariant`
 --> core/src/ptr/alignment.rs:3:52
  |
3 | use crate::ub_checks::{assert_unsafe_precondition, Invariant};
  |                                                                                         ^^^^^^^^^
  |
  = note: `-D unused-imports` implied by `-D warnings`
  = help: to override `-D warnings` add `#[allow(unused_imports)]`
```

Reply:
```suggestion
#[allow(unused_imports)]
```